crates/rooch/src/commands/account/commands/balance.rs

```diff
@@ -0,0 +1,139 @@
+// Copyright (c) The Starcoin Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::cli_types::{CommandAction, WalletContextOptions};
+use clap::Parser;
+use move_core_types::account_address::AccountAddress;
+use moveos_types::access_path::AccessPath;
+use moveos_types::module_binding::{ModuleBinding, MoveFunctionCaller};
+use moveos_types::move_option::MoveOption;
+use moveos_types::object::ObjectID;
+use moveos_types::state::MoveStructState;
+use moveos_types::state_resolver::resource_tag_to_key;
+use moveos_types::transaction::FunctionCall;
+use moveos_types::tx_context::TxContext;
+use rooch_rpc_api::api::MAX_RESULT_LIMIT_USIZE;
+use rooch_rpc_api::jsonrpc_types::account_view::AccountInfoView;
+use rooch_rpc_api::jsonrpc_types::{AccountAddressView, AnnotatedCoinStoreView, StructTagView};
+use rooch_types::account::BalanceInfo;
+use rooch_types::error::{RoochError, RoochResult};
+use rooch_types::framework::coin::CoinModule;
+
+/// Show a account info, only the accounts managed by the current node are supported
+#[derive(Debug, Parser)]
+pub struct BalanceCommand {
+    #[clap(short = 'a', long = "address")]
+    /// The account's address to show balance, if absent, show the default active account.
+    address: Option<AccountAddressView>,
+
+    /// Struct name as `<ADDRESS>::<MODULE_ID>::<STRUCT_NAME><TypeParam1?, TypeParam2?>`
+    /// Example: `0x123::counter::Counter`, `0x123::counter::Box<0x123::counter::Counter>`
+    #[clap(long = "coin_type")]
+    /// The block number or block hash for get state, if absent, use latest block state_root.
```

Comment:
This line of code comment is probably a copying error

Reply:
> This line of code comment is probably a copying error

fixed